whisper/src/lib.rs

```diff
@@ -260,15 +281,7 @@ impl WhisperFile {
 }
 
 pub fn suggest_archive(file: &WhisperFile, interval: Interval, now: u32) -> Option<u32> {
```

Comment:
As soon as this is now a part of `WhisperFile` API, I suggest to remove this function.

Reply:
Do we need to be compatible with python API? If not, I agree